### ISSUE TITLE
How do I test the picture

### ISSUE THREAD

**Issue by pyziq** [JihongJu/keras-fcn#14]
I have completed the training of the model. So I want to test the picture. So I use the model.predict(x). I get the shape with(1,500,500,21) , but any element in shape is NaN。 So how do I test it.

**Comment by JihongJu** [comment]
@pyziq Could you please provide your code, and training log?

**Comment by pyziq** [comment]
@JihongJu 
Thank you for reply

The code is:

`from keras_fcn import FCN
from keras.preprocessing import image
from keras.models import load_model
import numpy as np

model = load_model('fcn_vgg16.h5')
img = image.load_img('cat.jpg', target_size=(500, 500))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
preds =model.predict(x)
print preds`

and get the answer like this:
![image](https://user-images.githubusercontent.com/20399697/27261425-37b40fe8-5475-11e7-9d86-5ca524872caf.png)

the traning logs:
I get the loss：1.92e-07 acc:0.7435 val_acc:0.7432
Is the loss too small?

**Comment by JihongJu** [comment]
@pyziq Hmm, that's interesting. Would you mind to share with me a copy of your trained model? Otherwise, it may take a few days for me to reproduce this bug. My email address is daniel.jihong.ju@gmail.com.

**Comment by pyziq** [comment]
@JihongJu 

I have sent the trained model.
Thank you very much!

**Comment by JihongJu** [comment]
@pyziq Sorry for the late reply. I just added a callback function to determine which layer the `nan`s arise. It seems `score_feat2` was the problem. I am still debugging what was happening. Maybe you can also debug for what's happening with the new added `keras_fcn.callbacks.CheckNumericsOps`

**Comment by pyziq** [comment]
Thank you for reply， I will try it

**Comment by JihongJu** [comment]
Hi @pyziq, it turns out that the problem was caused by the lack of an activation layer on top of the last output layer. Now it is fixed.
